draft-ietf-quic-transport.md

```diff
@@ -3674,7 +3707,37 @@ Token Length:
 
 Token:
 
-: The value of the token.
+: The value of the token that was previously provided in a Retry packet or
```

Comment:
You're right, this fell in during the ELH combination.  So Initial packets *are* different because of the Token fields, which means that only 0-RTT and Handshake have the same ELH format.  Are we sure it's worth not simply having an explicit list of fields for those two?  That makes me more inclined to just revert the ELH change entirely and make each long-header packet type stand alone.